micro-ui/web/micro-ui-internals/packages/modules/workbench/src/pages/employee/MDMSView.js

```
@@ -125,30 +117,98 @@
+  const createLocalizationCode = (fieldName, fieldValue) => {
+    const upperFieldName = fieldName.toUpperCase();
+    const transformedValue = (fieldValue || "").replace(/\s+/g, "").toUpperCase();
+    return tranformLocModuleName(`${rawSchemaCode}_${upperFieldName}_${transformedValue}`);
+  };
+
+  let localizationCodes = [];
+  if (data && data.data && localisableFields.length > 0) {
+    localizationCodes = localisableFields.map(field => createLocalizationCode(field.fieldPath, data.data[field.fieldPath]));
+  }
+
+  let locale=Digit.StoreData.getCurrentLanguage();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'locale' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)